portal/src/main/webapp/js/src/oql/oqlfilter.js

```diff
@@ -215,7 +215,7 @@ window.OQL = (function () {
 	    if (alt_cmd.constr_rel === "=") {
 		match = +(d_cna === alt_cmd.constr_val.toLowerCase());
 	    } else {
-		var integer_copy_number = {"amp":2, "gain":1, "hetloss":-1, "homdel":-2};
```

Comment:
Same, frontend version of this file exists

https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/shared/lib/oql/oqlfilter.ts

Reply:
Just deleted this file (oql directory is obsolete)